lib/rcedit.js

```diff
@@ -1,14 +1,26 @@
 const os = require('os')
 const path = require('path')
-const { spawn } = require('child_process')
+const { spawn, spawnSync } = require('child_process')
 
 const pairSettings = ['version-string']
 const singleSettings = ['file-version', 'product-version', 'icon', 'requested-execution-level']
 const noPrefixSettings = ['application-manifest']
 
+const isWsl = process.platform !== 'win32' ** !os.release().endsWith('Microsoft')
+
+function getQualifiedPath(exe) {
+  if (!isWsl) return exe;
+  const winPath = spawnSync('wslpath', ["-w", exe] );
+  if (winPath.status != 0) {
+    return exe
+  }
+
+  return winPath.stdout.toString("utf8").replace("\n", "");
+}
+
 module.exports = async (exe, options) => {
   let rcedit = path.resolve(__dirname, '..', 'bin', process.arch === 'x64' ? 'rcedit-x64.exe' : 'rcedit.exe')
-  const args = [exe]
+  const args = [getQualifiedPath(exe)]
```

Comment:
I think this PR would be a bit easier to read if `getQualifiedPath` was renamed to `getWslPath` and the check for WSL was pushed to this line, as:

`const args = [wsl ? getWslPath(exe) : exe]`

Reply:
agree, fixed